src/system-health/health_checker/system_health_monitoring_config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "services_to_ignore": [],
+  "devices_to_ignore": [],
+  "external_checkers": [],
+  "polling_interval": 60,
+  "led_color": {
+    "fault": "amber",
+    "normal": "green",
+    "booting": "orange_blink"
+  }
+}

```

Comment:
Is this a sample configuration file or the default configuration file?

Reply:
It is a sample configuration file.